pyproject.toml

```diff
@@ -36,14 +36,13 @@ dependencies = [
 
 [project.optional-dependencies]
 rich = ["rich>=13.9.4"]
-cli = ["typer>=0.12.4", "python-dotenv>=1.0.0"]
+cli = ["typer>=0.16.0", "python-dotenv>=1.0.0"]
```

Comment:
I had to bump typer because click broke it recently.

I don't think bumping this package is an issue given that people can usually easily bump this without coding changes on their applications.

Reply:
pydantic or starlette would be more concerning.